### PR TITLE
Add Local Icon

### LINKED ISSUE #9202: Local
### Brand Name

Flywheel

### Website

https://localwp.com/

### Popularity Metric

The Similarweb rank is 152,120. See https://www.similarweb.com/website/localwp.com/

### Official Resources for Icon and Color

Example:
png: https://localwp.com/wp-content/uploads/2021/08/favicon-1.png
color: #51bb7b

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by nirajgiriXD**
**Issue:** closes #9202

**Similarweb rank:**
Global rank: [152,120](https://www.similarweb.com/website/localwp.com/)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Added `local` logo  in icons folder and updated `simple-icons.json` file with icon's data.

### Preview
local-icon-preview

### CODE REVIEW

File: icons/local.svg [hunk 1]
```diff
@@ -0,0 +1 @@
... m 7.6463 9.3931 l -7.6599 7.7655 v -2.069 l 7.0612 -7.1724 a 4.5159 4.4544 90 0 0 1.102 -1.6966 c 0.3401 0.4552 0.5034 0.8828 0.5034 1.2966 c -0.0136 0.5793 -0.3401 1.1862 -1.0068 1.8759 z m -4.6259 -1.3103 h -10.1088 a 0.5241 0.517 90 0 1 -0.517 -0.5241 v -0.0552 a 0.5241 0.517 90 0 1 0.517 -0.5241 h 10.1088 a 0.5241 0.517 90 0 1 0.517 0.5241 v 0.0552 a 0.5324 0.5252 90 0 1 -0.517 0.5241 z m -1.4422 -2.5793 v 0.0552 a 0.5241 0.517 90 0 1 -0.517 0.5241 h -6.2041 a 0.5241 0.517 90 0 1 -0.517 -0.5241 v -0.0552 a 0.5241 0.517 90 0 1 0.517 -0.5241 h 6.2041 a 0.5241 0.517 90 0 1 0.517 0.5241 z m -1.9184 -2.0138 v 0.0552 a 0.5241 0.517 90 0 1 -0.517 0.5241 h -2.4626 a 0.5241 0.517 90 0 1 -0.517 -0.5241 v -0.0552 a 0.5241 0.517 90 0 1 0.517 -0.5241 h 2.4626 a 0.5159 0.5088 90 0 1 0.517 0.5241 z m -0.517 7.5448 a 0.5241 0.517 90 0 1 0.517 0.5241 v 0.0552 a 0.5241 0.517 90 0 1 -0.517 0.5241 h -2.4626 a 0.5241 0.517 90 0 1 -0.517 -0.5241 v -0.0552 a 0.5241 0.517 90 0 1 0.517 -0.5241 z m 1.9184 -0.9241 h -6.2041 a 0.5241 0.517 90 0 1 -0.517 -0.5241 v -0.0552 a 0.5241 0.517 90 0 1 0.517 -0.5241 h 6.2041 a 0.5241 0.517 90 0 1 0.517 0.5241 v 0.0552 a 0.5241 0.517 90 0 1 -0.517 0.5241 z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/local.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Local</title><path d="m4.49 11.97 6.682-6.681a.638.638 0 0 0 .204-.476V.838a.7.7 0 0 0-.42-.624.68.68 0 0 0-.736.148L1.4 9.193c-.94.94-1.388 1.85-1.4 2.805s.434 1.85 1.36 2.774l8.858 8.86a.638.638 0 0 0 .476.203.39.39 0 0 0 .26-.082.68.68 0 0 0 .42-.626v-4a.692.692 0 0 0-.204-.476L4.489 11.97h.002zm-2.64 1.32c-.34-.45-.502-.872-.502-1.28.012-.57.34-1.182 1.007-1.85l7.66-7.662v2.057l-7.06 7.06A4.355 4.355 0 0 0 1.85 13.29zm8.166 8.205-6.451-6.45a.748.748 0 0 0-.094-.12c-.204-.207-.816-.819.094-1.961l6.45 6.449v2.082zM13.782.376a.668.668 0 0 0-.734-.15.68.68 0 0 0-.422.626v4.015c.004.18.076.35.204.476l6.681 6.68-6.681 6.682a.638.638 0 0 0-.204.476v3.96a.682.682 0 0 0 1.156.49l8.817-8.817c.94-.94 1.389-1.85 1.4-2.804.017-.952-.433-1.85-1.36-2.775L13.782.376zm.204 4.205V2.5l6.451 6.448c.026.044.06.084.094.122.204.204.816.817-.094 1.96l-6.449-6.45-.002.002zm7.647 9.267-7.66 7.661v-2.04l7.06-7.077a4.451 4.451 0 0 0 1.104-1.674c.34.45.504.872.504 1.28-.014.57-.34 1.17-1.008 1.85zm-4.626-1.294H6.9a.516.516 0 0 1-.516-.516v-.054c0-.286.23-.518.516-.518h10.11a.52.52 0 0 1 .518.518v.054a.526.526 0 0 1-.518.516h-.004zm-1.44-2.544v.056a.516.516 0 0 1-.52.516H8.842a.516.516 0 0 1-.518-.516v-.056c0-.285.232-.517.518-.517h6.205c.286 0 .516.232.516.517h.002zm-1.92-1.987v.054a.516.516 0 0 1-.517.518h-2.464a.516.516 0 0 1-.516-.518v-.054c0-.286.232-.516.516-.516h2.464a.508.508 0 0 1 .516.516zm-.517 7.443c.284 0 .516.232.516.518v.054a.516.516 0 0 1-.516.516h-2.464a.516.516 0 0 1-.516-.516v-.054c0-.286.232-.518.516-.518h2.464zm1.918-.912H8.843a.516.516 0 0 1-.518-.516v-.054a.52.52 0 0 1 .518-.518h6.205c.286 0 .516.232.516.518v.054a.516.516 0 0 1-.516.516z"/></svg>
```
This should fix the issues.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -7095,6 +7095,11 @@
             "hex": "10B146",
             "source": "https://lmms.io/branding"
         },
+        {
+            "title": "Local",
+            "hex": "51BB7B",
+            "source": "https://localwp.com/wp-content/uploads/2021/08/favicon-1.png"
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:7101]:
```suggestion
            "source": "https://localwp.com"
```
Do not link PNG. The icon is already available as SVG in the header of the front page.

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

**Review by PratyushJayachandran — APPROVED** [review]
Errors seem to be fixed. I haven't checked with source though.

**Comment by oohsai** [comment]
Hey! Glad if you help me. How did you perform the "SVG viewbox is 0 0 24 24' check? I do have the svg file with me already

**Comment by PratyushJayachandran** [comment]
> Hey! Glad if you could help me. How did you perform the "SVG viewbox is 0 0 24 24' check? I do have the svg file with me already
I just set the canvas size to 24x24 then center the icon.
`npm run lint` for running tests

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @nirajgiriXD 🎉

**Pull request merged**